dna/modify_test.go

```diff
@@ -70,6 +72,20 @@ func TestDel(t *testing.T) {
 	}
 }
 
+func TestDelStable(t *testing.T) {
+	destSeq := []Base{A, C, G, T}
+	origFirstBase := destSeq[0:1]
+	expectedSeq := []Base{A, G, T}
+	err := DeleteStable(&destSeq, 1, 2)
```

Comment:
Can you remind me why we would use "&destSeq" instead of just destSeq here? I understand you are trying to do things with memory address here but I'm not sure much past that.

Reply:
Maybe because this is an error and we don't want to actually change the original variable?

Reply:
DeleteStable takes a pointer to a slice of bases so as the first argument, so that has to be the format. destSeq is a slice of bases here and so DeleteStable is fed a pointer to it.

Reply:
Yay! This helped me so much Christi. And you read my mind about lines 80-81, I've been stuck there trying to wrap my head around it.